hal/inc/ble_hal.h

```diff
@@ -100,6 +100,12 @@ typedef enum hal_ble_scan_fp_t {
                                                                    address is a resolvable private address that cannot be resolved. */
 } hal_ble_scan_fp_t;
 
+typedef enum hal_ble_scan_phys_t {
```

Comment:
The advertiser will also use these enum values. Let's rename it to be more generic, for example: `hal_ble_phys_t`

Reply:
Good idea! done.